pallets/capacity/src/weights.rs

```diff
@@ -280,32 +266,19 @@ impl WeightInfo for () {
 	/// Proof: `Capacity::ProviderBoostRewardPools` (`max_values`: None, `max_size`: Some(113), added: 2588, mode: `MaxEncodedLen`)
 	fn start_new_reward_era_if_needed() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `193`
-		//  Estimated: `5063`
-		// Minimum execution time: 5_000_000 picoseconds.
-		Weight::from_parts(5_000_000, 5063)
-			.saturating_add(RocksDbWeight::get().reads(2_u64))
-			.saturating_add(RocksDbWeight::get().writes(1_u64))
-	}
-	/// Storage: `Capacity::CurrentEpochInfo` (r:1 w:1)
-	/// Proof: `Capacity::CurrentEpochInfo` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
-	/// Storage: `Capacity::EpochLength` (r:1 w:0)
-	/// Proof: `Capacity::EpochLength` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
-	fn start_new_epoch_if_needed() -> Weight {
-		// Proof Size summary in bytes:
-		//  Measured:  `0`
-		//  Estimated: `2974`
-		// Minimum execution time: 2_231_000 picoseconds.
-		Weight::from_parts(2_419_000, 2974)
+		//  Measured:  `229`
+		//  Estimated: `4073`
+		// Minimum execution time: 6_626_000 picoseconds.
+		Weight::from_parts(7_003_000, 4073)
 			.saturating_add(RocksDbWeight::get().reads(2_u64))
 			.saturating_add(RocksDbWeight::get().writes(1_u64))
 	}
 	/// Storage: `Capacity::ProviderBoostHistories` (r:1 w:1)
 	/// Proof: `Capacity::ProviderBoostHistories` (`max_values`: None, `max_size`: Some(641), added: 3116, mode: `MaxEncodedLen`)
 	/// Storage: `Capacity::StakingAccountLedger` (r:1 w:1)
 	/// Proof: `Capacity::StakingAccountLedger` (`max_values`: None, `max_size`: Some(57), added: 2532, mode: `MaxEncodedLen`)
-	/// Storage: `Capacity::StakingRewardPool` (r:1 w:1)
-	/// Proof: `Capacity::StakingRewardPool` (`max_values`: None, `max_size`: Some(60), added: 2535, mode: `MaxEncodedLen`)
+	/// Storage: `Capacity::CurrentEraProviderBoostTotal` (r:1 w:1)
+	/// Proof: `Capacity::CurrentEraProviderBoostTotal` (`max_values`: Some(1), `max_size`: Some(16), added: 511, mode: `MaxEncodedLen`)
```

Comment:
@shannonwells would you mind take a look at these changes?

Reply:
The diff is a little weird, so I checked out the branch, it looks right.